websites/A/Armorgames/presence.ts

```diff
@@ -0,0 +1,112 @@
+      title = document.querySelector("#main > h2");
+      if (title.textContent == "Your Friends") {
+        presenceData.details = `Viewing their friends`;
+        presenceData.buttons = [{label: "View Friends", url: window.location.href}];
+      } else {
+        presenceData.details = `Viewing the friends of:`;
+        presenceData.state = title.textContent;
+        presenceData.buttons = [{label: "View Friends", url: window.location.href}];
+      }
+    } else if (page.includes("/user/")) {
+      title = document.querySelector("#user-data > h1");
+      presenceData.details = `Viewing the profile of:`;
+      presenceData.state = title.textContent;
+      presenceData.buttons = [{label: "View Profile", url: window.location.href}];
+    } else if (page.includes("/author/")) {
+      title = document.querySelector("#categorylisting > h2 > a");
+      presenceData.details = `Viewing Games Made By:`;
+      presenceData.state = title.textContent;
+      presenceData.buttons = [{label: "View Profile", url: window.location.href}];
+    } else if (page == "/register" || page == "/register/") {
+      presenceData.details = "Registering";
+    } else if (page == "/settings/general") {
+      presenceData.details = "Viewing:";
+      presenceData.state = "General Settings";
+    } else if (page == "/settings/friends") {
+      presenceData.details = "Viewing:";
+      presenceData.state = "Their Friends";
+    } else if (page == "/settings/quests") {
+      presenceData.details = "Viewing:";
+      presenceData.state = "Quests";
+    } else if (page == "/settings/favs") {
+      presenceData.details = "Viewing:";
+      presenceData.state = "Favs";
+    } else if (page == "/settings/contact") {
+      presenceData.details = "Viewing:";
+      presenceData.state = "Contact Settings";
+    } else if (page == "/settings/accounts") {
+      presenceData.details = "Viewing:";
+      presenceData.state = "Soicial Settings";
+    } else if (page == "/settings/purchases") {
+      presenceData.details = "Viewing:";
+      presenceData.state = "My Purchases";
+    } else if (page == "/settings/password") {
+      presenceData.details = "Viewing:";
+      presenceData.state = "Password Settings";
+    } else if (page.includes("/games/")) {
+      title = document.querySelector("#content-canvas > section.game-header.clearfix > h1");
+      presenceData.details = "Browsing:";
+      presenceData.state = "All Games";
+      presenceData.buttons = [{label: "Browse Games", url: window.location.href}];
+    } else if (page.includes("game")) {
+      title = document.querySelector("#content-canvas > section.game-header.clearfix > h1");
+      presenceData.details = "Playing:";
+      presenceData.state = title.textContent;
+      presenceData.buttons = [{label: "Play Game", url: window.location.href}];
+    } else {
+      presenceData.details = "idk this page.";
+      presenceData.state = "Create an issue on github";
+    }
```

Comment:
Idk this page! Literally shitty and unnecessary thing. If you want to show user that they are on page that doesn't exist it should look at this:
```suggestion
    } else {
      presenceData.details = "Page Not Found";
    }
```
and you can error it in console by `presence.error` if I'm correct.